ash/services.py

```diff
@@ -22,10 +23,13 @@ def get_running_services(cls) -> list[str]:
         if cls.client is None:
             return result
 
+        logging.debug("Checking for Running services.")
```

Comment:
Please remove log statements, which are executed in every iteration as they flood the logs

Reply:
particularly lines 26, 32, 95, 113